assets/app/views/browse/routes.html

```diff
@@ -0,0 +1,42 @@
+<div ng-controller="ProjectController" class="content">
+  <project-page>
+    <div ng-controller="RoutesController">
+      <div class="page-header page-header-bleed-right">
+        <h1>Routes</h1>
+      </div>
+      <alerts alerts="alerts"></alerts>
+      <div class="table-responsive">
+        <table class="table table-bordered table-hover">
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th>Hostname</th>
+              <th>Routes to</th>
+              <th>Age</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr ng-if="(routes | hashSize) == 0"><td colspan="6"><em>{{emptyMessage}}</em></td></tr>
+            <tr ng-repeat="route in routes | orderObjectsByDate : true">
+              <td><a href="project/{{projectName}}/browse/routes/{{route.metadata.name}}">{{route.metadata.name}}</a></td>
+              <td>
+                <span ng-if="(route | isWebRoute)">
+                  <a href="{{route | routeWebURL}}" target="_blank">{{route | routeLabel}}</a>
+                </span>
+                <span ng-if="!(route | isWebRoute)">
+                  {{route | routeLabel}}
+                </span>
+              </td>
+              <td>
+                <span>{{route.spec.to.kind}}: </span>
+                <span ng-if="route.spec.to.kind != 'Service'">{{route.spec.to.name}}</span>
+                <span ng-if="route.spec.to.kind == 'Service'"><a href="project/{{projectName}}/browse/services/{{route.spec.to.name}}">{{route.spec.to.name}}</a></span>
+              </td>
+              <td><relative-timestamp timestamp="route.metadata.creationTimestamp" drop-suffix="true"></relative-timestamp></td>
```

Comment:
I'm not sure how useful creation time is in these tables. I realize we have it in other places.